commontype/fee_config.go

```diff
@@ -81,6 +82,22 @@ func (f *FeeConfig) Verify() error {
 	return f.checkByteLens()
 }
 
+// Equal checks if given [other] is same with this FeeConfig.
+func (f *FeeConfig) Equal(other *FeeConfig) bool {
```

Comment:
Can we add unit tests for each of these cases as well?

Reply:
added